roles/ovirt-host-deploy-ovirt-metrics/README.md

```diff
@@ -0,0 +1,35 @@
+oVirt host deploy - oVirt metrics
```

Comment:
I think we don't need it if there is no variables to document.

Reply:
Don't need the entire file? @machacekondra